osbs/tekton.py

```diff
@@ -52,6 +54,12 @@ def check_response(response, log_level=logging.ERROR):
         raise OsbsResponseException(message=content, status_code=response.status_code)
 
 
+def get_sorted_task_runs(task_runs: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I think this returns a list of tuples
```python
In [1]: d = {'a': 1, 'b': 2}
In [2]: print(sorted(d.items()))
[('a', 1), ('b', 2)]
```
```suggestion
def get_sorted_task_runs(task_runs: Dict[str, Any]) -> List[Tuple[str, Dict[str, Any]]]:
```

Reply:
fixed